components/consentRequestForm/index.jsx

```diff
@@ -33,36 +33,105 @@ import InfoTooltip from "../infoTooltip";
 import RequestSection from "./requestSection";
 import styles from "./styles";
 import { mockApp } from "../../__testUtils/mockApp";
+import {
+  getPurposeString,
+  getPurposeUrl,
+  getRequestedAccesses,
+} from "../../src/models/consent/request";
+import ConfirmationDialogContext from "../../src/contexts/confirmationDialogContext";
+import ConfirmationDialog from "../confirmationDialog";
 
 const useStyles = makeStyles((theme) => createStyles(styles(theme)));
 
+const NO_ACCESS_DIALOG_TITLE = "Your haven't selected any access";
+export const CONSENT_REQUEST_NO_ACCESS_DIALOG =
+  "consent-request-no-access-dialog";
+export const TESTCAFE_ID_CONSENT_REQUEST_SUBMIT_BUTTON =
+  "consent-request-submit-button";
+
 export default function ConsentRequestFrom() {
   const classes = useStyles();
   const bem = useBem(classes);
   const { consentRequest } = useContext(ConsentRequestContext);
+  const [selectedAccess, setSelectedAccess] = useState([]);
   // FIXME: using a mock for the app profile - we will fetch profile later
   const agentProfile = mockApp();
   const agentName = getStringNoLocale(agentProfile, foaf.name);
-  const purposeUrl =
-    consentRequest?.credentialSubject?.hasConsent[0].forPurpose; // getting the first in this array for now
+  const purposeUrl = getPurposeUrl(consentRequest);
   // FIXME: we will later fetch the description from the purpose Url
-  const purposeDescription = "Some Specific Purpose";
+  const purposeDescription = getPurposeString();
+  const {
+    confirmed,
+    open,
+    setContent,
+    setOpen,
+    setTitle,
+    closeDialog,
+  } = useContext(ConfirmationDialogContext);
+  const [confirmationSetup, setConfirmationSetup] = useState(false);
+
+  const handleSubmit = (e) => {
+    e.preventDefault();
+    if (selectedAccess.length) {
+      setConfirmationSetup(false);
+      return;
+    }
+    setConfirmationSetup(true);
+    setOpen(CONSENT_REQUEST_NO_ACCESS_DIALOG);
+    setTitle(NO_ACCESS_DIALOG_TITLE);
+    setContent(`${agentName} will not have access to anything in your Pod`);
```

Comment:
Do we have a way of setting the main button copy? I'd like to change it from "Confirm" to "Continue with no access", as in https://www.sketch.com/s/f073950c-60f8-47d1-9d8f-4824de19ba08/a/jgM1JKz 

Also can you please add a period at the end of "${agentName} will not have access to anything in your Pod" 🙃

Reply:
done :) sorry I missed that!